src/substrate-service.ts

```diff
@@ -66,6 +66,37 @@ export class SubstrateService {
     });
   }
 
+  public async issueAssetToUserBatch(destinationAccountPublicKey: string, amount: string, key_2: string, key_3: string): Promise<IssueAssetResponse> {
```

Comment:
Let this function will accept the array of keys, so it is flexible on amount if items in batch to process

Reply:
Resolved and changed per request